e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js

```diff
@@ -288,6 +288,36 @@ describe('I18456 Built-in slash commands: common', () => {
         cy.uiWaitUntilMessagePostedIncludes(`You were added to the channel by @${user1.username}`);
     });
 
+    it('MM-T661 /invite extra white space before @ in DM or GM', () => {
+        const user = userGroup[6];
+        const userToInviteGM = userGroup[5];
+        const userToInviteDM = userGroup[4];
+
+        cy.apiAddUserToChannel(testChannel.id, user.id);
+        loginAndVisitDefaultChannel(user, `${team1.name}/channels/${testChannel.name}`);
+        cy.get('#postListContent', {timeout: TIMEOUTS.HALF_MIN}).should('be.visible');
+
+        // # In a GM use the /invite command to invite a user to a channel you have permission to add them to but place extra white space before the username
+        cy.postMessage(`/groupmsg @${userGroup[0].username} @${userGroup[1].username}`);
+        cy.postMessage(`/invite        @${userToInviteGM.username} ~${testChannel.name}`);
+
+        // * User added to channel as expected
+        cy.uiWaitUntilMessagePostedIncludes(`${userToInviteGM.username} added to ${testChannel.name} channel.`);
+
+        cy.get('#addDirectChannel').click();
+        cy.get('#selectItems').type(`${userToInviteDM.username}`);
+        cy.wait(TIMEOUTS.ONE_SEC);
```

Comment:
Could someone give me a better suggestion than waiting 1 second?  If I don't insert this wait, cypress tells me the DOM has been detached.  Visually the result of the search appears, disappears and reappears.

Reply:
One way is to do other assertion, like:
```javascript
cy.findByText('Loading').should('be.visible');
cy.findByText('Loading').should('not.exist');
```